### ISSUE TITLE
Fix UiT collection codes

### ISSUE THREAD

**Issue by rukayaj** [gbif-norway/helpdesk#62]
MUSIT for some reason is publishing the UiT collections in under one or two generalised collection codes (EVERT and TROM) instead of the collection codes that they actually use. We also need to sync these collection codes with grscicoll, there are also some unpublished collections which should still get listed.

Zoological collections list: https://uit.no/research/natscicol/project?pid=717007

Herbarium collections list: https://uit.no/research/natscicol/project?pid=717010

Geology collections list: https://uit.no/research/natscicol/project?pid=717009 (not yet in grscicoll - should we add the fossils one at least? does grscicoll aim to cover all scientific collections or just biological?)

Quick links to the API facet for collection codes for each uit dataset so we can check them easily:
[gbif](https://gbif.org/dataset/b581fcfa-9f31-431a-8431-1800ef8a554b)	 / 	[b581fcfa-9f31-431a-8431-1800ef8a554b](https://api.gbif.org/v1/occurrence/search?facet=collectionCode&limit=0&datasetKey=b581fcfa-9f31-431a-8431-1800ef8a554b)
[gbif](https://gbif.org/dataset/77e91a4f-7b9d-458d-b585-298aa1ef41f8)	 / 	[77e91a4f-7b9d-458d-b585-298aa1ef41f8](https://api.gbif.org/v1/occurrence/search?facet=collectionCode&limit=0&datasetKey=77e91a4f-7b9d-458d-b585-298aa1ef41f8)
[gbif](https://gbif.org/dataset/02f57aa9-12ae-4c8c-9d9f-f8b3245cf31a)	 / 	[02f57aa9-12ae-4c8c-9d9f-f8b3245cf31a](https://api.gbif.org/v1/occurrence/search?facet=collectionCode&limit=0&datasetKey=02f57aa9-12ae-4c8c-9d9f-f8b3245cf31a)
[gbif](https://gbif.org/dataset/060eecc9-75b0-4952-bb48-e400d3ad7771)	 / 	[060eecc9-75b0-4952-bb48-e400d3ad7771](https://api.gbif.org/v1/occurrence/search?facet=collectionCode&limit=0&datasetKey=060eecc9-75b0-4952-bb48-e400d3ad7771)
[gbif](https://gbif.org/dataset/a33a6578-bf83-4fa9-b39b-3a3738b72b68)	 / 	[a33a6578-bf83-4fa9-b39b-3a3738b72b68](https://api.gbif.org/v1/occurrence/search?facet=collectionCode&limit=0&datasetKey=a33a6578-bf83-4fa9-b39b-3a3738b72b68)
[gbif](https://gbif.org/dataset/b4804f19-8a8a-49e7-8dc2-79b528635696)	 / 	[b4804f19-8a8a-49e7-8dc2-79b528635696](https://api.gbif.org/v1/occurrence/search?facet=collectionCode&limit=0&datasetKey=b4804f19-8a8a-49e7-8dc2-79b528635696)
[gbif](https://gbif.org/dataset/d391c193-0fc0-4a96-bdac-6043dd9516d1)	 / 	[d391c193-0fc0-4a96-bdac-6043dd9516d1](https://api.gbif.org/v1/occurrence/search?facet=collectionCode&limit=0&datasetKey=d391c193-0fc0-4a96-bdac-6043dd9516d1)
[gbif](https://gbif.org/dataset/8dbe67d7-26df-4810-a989-d246f0e93edb)	 / 	[8dbe67d7-26df-4810-a989-d246f0e93edb](https://api.gbif.org/v1/occurrence/search?facet=collectionCode&limit=0&datasetKey=8dbe67d7-26df-4810-a989-d246f0e93edb)
[gbif](https://gbif.org/dataset/e92bb2ed-1efe-4d2a-a17d-52182c562db8)	 / 	[e92bb2ed-1efe-4d2a-a17d-52182c562db8](https://api.gbif.org/v1/occurrence/search?facet=collectionCode&limit=0&datasetKey=e92bb2ed-1efe-4d2a-a17d-52182c562db8)
[gbif](https://gbif.org/dataset/b5a4d944-8a51-4104-a4a7-c3637128bcc7)	 / 	[b5a4d944-8a51-4104-a4a7-c3637128bcc7](https://api.gbif.org/v1/occurrence/search?facet=collectionCode&limit=0&datasetKey=b5a4d944-8a51-4104-a4a7-c3637128bcc7)
[gbif](https://gbif.org/dataset/88fd0226-d867-4946-be8b-31f584f2201d)	 / 	[88fd0226-d867-4946-be8b-31f584f2201d](https://api.gbif.org/v1/occurrence/search?facet=collectionCode&limit=0&datasetKey=88fd0226-d867-4946-be8b-31f584f2201d)
[gbif](https://gbif.org/dataset/f2a77c80-1e74-4c23-a3c9-c52cede89434)	 / 	[f2a77c80-1e74-4c23-a3c9-c52cede89434](https://api.gbif.org/v1/occurrence/search?facet=collectionCode&limit=0&datasetKey=f2a77c80-1e74-4c23-a3c9-c52cede89434)
[gbif](https://gbif.org/dataset/8d1253fc-30b6-4b4d-a12d-d711391e5382)	 / 	[8d1253fc-30b6-4b4d-a12d-d711391e5382](https://api.gbif.org/v1/occurrence/search?facet=collectionCode&limit=0&datasetKey=8d1253fc-30b6-4b4d-a12d-d711391e5382)
[gbif](https://gbif.org/dataset/0f061eff-6854-4bb3-abe2-acb184ea3ab7)	 / 	[0f061eff-6854-4bb3-abe2-acb184ea3ab7](https://api.gbif.org/v1/occurrence/search?facet=collectionCode&limit=0&datasetKey=0f061eff-6854-4bb3-abe2-acb184ea3ab7)
[gbif](https://gbif.org/dataset/374e0d4c-cf9f-4e1a-97a4-14123ee1bb7e)	 / 	[374e0d4c-cf9f-4e1a-97a4-14123ee1bb7e](https://api.gbif.org/v1/occurrence/search?facet=collectionCode&limit=0&datasetKey=374e0d4c-cf9f-4e1a-97a4-14123ee1bb7e)
[gbif](https://gbif.org/dataset/e87a12af-fc4c-4315-bff7-c7b827379aca)	 / 	[e87a12af-fc4c-4315-bff7-c7b827379aca](https://api.gbif.org/v1/occurrence/search?facet=collectionCode&limit=0&datasetKey=e87a12af-fc4c-4315-bff7-c7b827379aca)
[gbif](https://gbif.org/dataset/d0aa984e-c6d3-45ee-8fc0-df1df8f4126b)	 / 	[d0aa984e-c6d3-45ee-8fc0-df1df8f4126b](https://api.gbif.org/v1/occurrence/search?facet=collectionCode&limit=0&datasetKey=d0aa984e-c6d3-45ee-8fc0-df1df8f4126b)
[gbif](https://gbif.org/dataset/ed527c71-23aa-40cf-b16b-1a5b8ec6770a)	 / 	[ed527c71-23aa-40cf-b16b-1a5b8ec6770a](https://api.gbif.org/v1/occurrence/search?facet=collectionCode&limit=0&datasetKey=ed527c71-23aa-40cf-b16b-1a5b8ec6770a)
[gbif](https://gbif.org/dataset/4894f2f8-74b5-403e-bd8d-6fe5123a3f71)	 / 	[4894f2f8-74b5-403e-bd8d-6fe5123a3f71](https://api.gbif.org/v1/occurrence/search?facet=collectionCode&limit=0&datasetKey=4894f2f8-74b5-403e-bd8d-6fe5123a3f71)

**Comment by rukayaj** [comment]
Yes that's what I'm doing, I've added collectionCode (and also collectionID) for a few of them so far, I'll do the others tomorrow :)

**Comment by rukayaj** [comment]
I've finished adding the collections missing from grscicoll (the herbarium ones mostly), and adding the collectionIDs + proper collectionCodes to all the datasets. 

Some outstanding issues:

- Andreas and I have found one dataset which has recently got duplicate catalogNumbers which is causing a problem (TSZR). He's working on fixing that with MUSIT now. 
- The Bird + Bird egg collections still need to get added to grscicoll (waiting to hear what collection code they should get)
- It's not clear where the [arthropod](https://ipt.gbif.no/manage/resource?r=trom_arthropoda) and [entomology](https://ipt.gbif.no/manage/resource.do?r=trom_entomology) collections fit in in [the UiT collections page](https://uit.no/research/natscicol/project?pid=717007): 

Note: the occurrenceIDs are dwc triplets, but have remained the same (using the old collectionCodes)

**Comment by dagendresen** [comment]
duplicate catalogNumbers causing problems because ... the catalogNumbers are used as occurrenceID? Any thoughts/plans on assigning materialSampleIDs to the specimens?

**Comment by rukayaj** [comment]
Update: this is still in progress, waiting for MUSIT to finish changing their data publication views.

**Comment by rukayaj** [comment]
Just sent a follow up email.

**Comment by rukayaj** [comment]
All fine now.
